### ISSUE TITLE
Tweaking Emailing 

### ISSUE THREAD

**Issue by torwag** [ddvk/remarkable-hacks#58]
I believe that most of the points below are unlikely to be solved by hacking into xochitl as I assume they are features on the server side but anyhow I would like to list them here for discussions:
My biggest hassle with the email service are
1. Not being able to send more than one document in a single mail (assume server side issue)

2. The notification pops up quickly and disappears, always left me in the unsure state whether it was sent or not. A pop-up which requires confirmation, or even better a sort of "was-send" marking or a history (pop-up window with "was sent to: $email on  $date"-list would be awesome.

3. Being able to define a standard CC recipient (me) as standard, thus that I get a copy of the exact file which I send, both for backup (in case I do additional changes later) and to be able to see that it was really sent.

4. Email templating. Ok I admit this is science fiction but being able to compose templates e.g. with moustache references like {{Name}}. And being able to write into a (dynamically generated)  Notebook Template which ask me for those references like Name:______, using the OCR of the rM, would allow me to compose emails very quickly without using the keyboard at all.
E.g. 
Email-Template:----------------------------------
Dear {{Name}},
please find attached the signed document.
We have to submit this document until {{Deadline}}.
<if Notes !=Nil, I have the following comments on this:
{{Notes}}>
Please feel free to give me a call under XXXXX if you have further questions.
Best regards
------------------------------------------------------
Notebook-Template:-----------------------------
Name:
Deadline:
Notes:
-------------------------------------------------------

**Comment by ddvk** [comment]
nope, all is done on the device, but still not that easy to implement.

a history log is doable, still requires some effort
